test/configure.js

```diff
@@ -29,7 +29,13 @@ suite( 'Configure: Settings', () => {
         }, data => {
 
             assert.isArray( data, 'Sugarcoat returns an array when we are missing a dest.' );
-            assert.propertyVal( data[0], 'key', 'settings.dest', 'We have dest in the error array');
+
+            data.forEach( ( errorObj, index ) => {
+
+                assert.instanceOf( errorObj, Error, 'The object was an Error Object.' );
```

Comment:
Looks like we're now testing that there is an Error object. But we don't know which error was returned. Can we make this more specific to the test?